packages/create-plugin/src/utils/utils.templates.ts

```diff
@@ -128,8 +126,8 @@ export function getTemplateData(cliArgs?: GenerateCliArgs): TemplateData {
       isNPM: isNPM(packageManagerName),
       version: currentVersion,
       bundleGrafanaUI,
-      useReactRouterV6: shouldUseReactRouterV6(cliArgs.pluginType),
-      reactRouterVersion: getReactRouterVersion(cliArgs.pluginType),
+      useReactRouterV6: features.useReactRouterV6 || false,
```

Comment:
Should this be true to [match the default value](https://github.com/grafana/plugin-tools/blob/academo/update-scenes-app-template/packages/create-plugin/src/constants.ts#L47) of `features.useReactRouterV6`?